core/client/kt/verify.go

```diff
@@ -120,13 +121,23 @@ func (v *Verifier) VerifyGetEntryResponse(ctx context.Context, userID string, in
 	}
 	Vlog.Printf("✓ Signed Map Head signature verified.")
 
+	// Update the log root.
+	// TODO(gdbelvin): Gossip root.
+	if err := v.log.UpdateRoot(in.LogRoot, in.LogConsistency); err != nil {
```

Comment:
This call to `UpdateRoot` seems to me a bit misplaced in a function that verifies things. From its name, `VerifyGetEntryResponse` gives the impression that it's a passive function. It might be better for `VerifyGetEntryResponse` callers to call `UpdateRoot`. What do you think?